src/data/references.json

```diff
@@ -6406,5 +6406,29 @@
             "contents": ["attack"],
             "year": 2023
         }
+    },
+    {
+        "title": "Backdoor in upstream xz/liblzma leading to ssh server compromise (CVE-2024-3094)",
+        "link": "https://www.openwall.com/lists/oss-security/2024/03/29/4",
+        "vectors": [{
+            "avId": "AV-800",
+            "avName": "Become Maintainer"
+        }],
```

Comment:
I would scope the reference under the "contribute as maintainer" context as it was contributed in the source code repository. What do you think?
 {
                "avId": "AV-800",
                "avName": "Become Maintainer",
                "scopeAvId": "AV-302",
                "scopeAvName": "Contribute as Maintainer"
            }

Reply:
@serenaponta Makes sense, just added the scope